simple/java/extract-variable_before2.java

```diff
@@ -0,0 +1,10 @@
+void renderBanner() {
+  // When all the parameters are set, counted 
```

Comment:
Render banner only if we're in fullscreen mode and a change is requested either in frame or target, or experiment is active.